subprojects/docs/src/docs/release/notes.md

```diff
@@ -213,11 +226,32 @@ Thus, when making small incremental changes, the `--no-rebuild` command-line opt
 
 ## Fixed issues
 
+## Known issues
+
+Known issues are problems that were discovered post release that are directly related to changes made in this release.
+
 ### Reporting of TestNG classes/methods
 
 When using a recent version of TestNG (6.9.13.3 or newer), classes were reported to `TestListeners` as sibling `TestDescriptors` of test method `TestDescriptors`.
 Now, `TestDescriptors` of classes are parents of their enclosing method `TestDescriptors`.
 
+## Potential breaking changes
+
+This is a new major version of Gradle, so many of the things that were deprecated in the Gradle 4.x versions have been removed.
+For examples, there have been fixes to dependency resolution that break corner cases, and running Gradle now requires Java 8 or higher (though tests can be run using Java 6 or 7).
+Breaking changes are listed with detailed explanations in [upgrading Gradle 4.x](userguide/upgrading_version_4.html).
+
+It is recommended that you: 
```

Comment:
Since this is now the preamble of the upgrade guide, should we also remove this from the release notes?

Reply:
Removed, but I was tempted to duplicate it because I think people should really follow these instructions.